src/BitStringAddresses/fermifs.jl

```diff
@@ -131,6 +131,54 @@ Base.:(==)(a::FermiFS, b::FermiFS) = a.bs == b.bs
 num_occupied_modes(::FermiFS{N}) where {N} = N
 occupied_modes(a::FermiFS{N,<:Any,S}) where {N,S} = FermiOccupiedModes{N,S}(a.bs)
 
+num_unoccupied_modes(::FermiFS{N,M}) where {N,M} = M - N
+function unoccupied_modes(a::FermiFS{N,M,S}) where {N,M,S}
+    s = ~a.bs
```

Comment:
I'm not sure I understand why the bitstring needs to be inverted here. Can't you just search for zeros in `iterate` instead of searching for ones in the inverted bitstring? I'm pointing this out because it's not obvious later in the code that the bistring stored as `storage` in `FermiUnoccupiedModes` is inverted.

If searching for ones is easier than searching for zeros, you can also invert the bitstring in `iterate` itself. Inverting an integer is almost free on modern processors and shouldn't affect performance.